nrf_security/doc/nrf_security.rst

```diff
@@ -0,0 +1,522 @@
+.. _nrf_security_readme:
+
+Nordic security module
+######################
+
+The nrf_security module provides an integration between mbed TLS and software libraries that provide hardware-accelerated cryptographic functionality on selected Nordic Semiconductor SoCs.
+This module includes an mbed TLS glue layer to enable both hardware-accelerated and standard mbed TLS software implementation at the same time.
+
+.. note::
+   The nrf_security module interfaces with the :ref:`nrf_cc310_mbedcrypto_readme`.
+   This library conforms to a specific version of mbed TLS.
+
+
+Prerequisites
+=============
+The nrf_security module requires mbed TLS. It checks for the presence of mbed TLS in the following ways:
+
+* It looks for an environmental variable ``ARM_MBEDTLS_PATH``.
+* If west is used for building, it tries to locate mbed TLS during the build.
```

Comment:
I think I would swap those two lines and rephrase a little.
```
* CMake uses `west` as part of the build process to locate mbed TLS.
* ``ARM_MBEDTLS_PATH`` can be set to enforce a specific path to mbed TLS or in case `west` is not available
```

Reply:
Accepted